src/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+  background-color: #eee;
+  color: #555;
+}
+
+#content {
+  background-color: #fff;
+  width: 50%;
+  margin: auto;
+  box-shadow: 5px 5px #ddd;
+}
+
+#content ul {
+  padding: 0;
+  margin: 0 auto;
+}
+
+.listItem {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid #ddd;
+  height: 50px;
+  padding: 0 20px;
+}
+
+.listItem input {
+  border: none;
+  outline: none;
+}
+
+.listItem input:placeholder-shown {
+  font-style: italic;
+}
+
+.listItem img {
+  width: 20px;
+}
+
+#addNew {
+  font-style: italic;
+  color: #888;
+  margin-bottom: 0;
+}
+
+#addNew div {
+  outline: none;
+}
+
+.checkLabel {
+  outline: none;
+}
+
+#enterIcon {
+  width: 12px;
+}
+
+.check {
+  margin-right: 10px;
+}
+
+.check:checked + .checkLabel {
+  text-decoration: line-through;
+  color: #888;
+}
+
+.clearButton {
+  width: 100%;
+  height: 60px;
+  border: none;
+  color: #888;
+  font-size: 17px;
```

Comment:
How about using percentages instead of fixed pixels to help with easier responsive design of the page